src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  font-weight: 400;
+
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  &__product-img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: auto;
+    margin-bottom: 40px;
+  }
+
+  &__items-container {
+    color: $main-font-color;
+  }
+
+  &__product-name {
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-font-color;
+  }
+
+  &__rate-row,
+  &__price-row {
+    @include flex;
+    align-items: flex-end;
+  }
+
+  &__rate-row {
+    margin-bottom: 24px;
+  }
+
+  &__rewies-counter {
+    font-size: 10px;
+    line-height: 16px;
+  }
+
+  &__price-row {
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+    color: $secondary-font-color;
+  }
+
+  &_price-value {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+  }
+
+  &__button {
+    display: block;
+    height: 40px;
+
+    font-size: 14px;
+    line-height: 40px;
+    font-weight: 700;
+    text-decoration: none;
+    text-align: center;
+    color: $card-bgc;
+
+    background-color: $blue-color;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    transition: 0.3s;
```

Comment:
It is better to clarify for which properties you want to apply the transition.
https://developer.mozilla.org/en-US/docs/Web/CSS/transition
Also, transition time should be the same for the whole page, so it's better to add a variable for it.